packages/peer-store/src/utils/to-peer-pb.ts

```diff
@@ -133,7 +134,7 @@ export async function toPeerPB (peerId: PeerId, data: Partial<PeerData>, strateg
   }
 
   const output: PeerPB = {
-    addresses: await dedupeFilterAndSortAddresses(peerId, options.addressFilter ?? (async () => true), addresses),
+    addresses: await dedupeFilterAndSortAddresses(peerId, options.addressFilter ?? (async () => true), addressToFqMultiaddr(peerId, addresses)),
```

Comment:
as far as I could tell, this `to-peer-pb` is called prior to saving/merging/etc.. so it seemed like the best place to add this if we _do_ want the store to have the peer Id. If we don't, either src/index or src/store would be the best place. lmk.

Reply:
I don't think we should store the multiaddrs with peer ids in the datastore, the reason being some users will have existing datastore which will then have some peers with multiaddrs with peer ids and some without, so we'd need to do a datastore migration to get everything consistent.

Instead we should have a firm disctinction between the types `Peer` and `PeerInfo`

`peerStore.get` returns `Peer`, this is the fully-featured version with peer ids appended to the multiaddrs (which we would do prior to returning them), along with tags, metadata, etc, then add a new `peerStore.getInfo` method that returns a [PeerInfo](https://docs.libp2p.io/concepts/fundamentals/peers/#peer-info) - this just has the peer id and the multiaddrs without the peer id and is the structure you'd expect to exchange with other implementations.